dojo/api_v2/views.py

```diff
@@ -257,9 +258,11 @@ def perform_update(self, serializer):
     def get_queryset(self):
         if not self.request.user.is_staff:
             findings = Finding.objects.filter(
-                test__engagement__product__authorized_users__in=[self.request.user])
+                test__engagement__product__authorized_users__in=[self.request.user]).prefetch_related(
+                    'endpoints', 'reviewers', 'images', 'found_by', 'notes', 'risk_acceptance_set')
         else:
-            findings = Finding.objects.all()
+            findings = Finding.objects.all().prefetch_related('endpoints', 'reviewers', 'images', 
+                                                              'found_by', 'notes', 'risk_acceptance_set')
         return findings.prefetch_related('test',
                                         'test__test_type',
                                         'test__engagement',
```

Comment:
could it be better to add them here (and only once)?

Reply:
I followed your advise. I also removed queryset because if I am correct it is overwritten by get_queryset anyway?